docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
+
+This is targeted at Docker 1.7.
+
+# Usage
+
+Plugins can be loaded using a special `docker plugin load` command, as follows:
+
+```
+$ docker plugin load -e $ARGS clusterhq/flocker-plugin $@
```

Comment:
why should this have `-e $ARGS` and `$@`? I suppose these are placeholders for the launch configuration... The line is a little confusing, I think if it just said `docker plugin load clusterhq/flocker-plugin`, it'd totally fine. You can hint about configuration though, which if you want can be through arguments. Although, keeping configuration out of the scope for now would be good, you could suggest that configuration would be part of the image if you like.